File: BOJ/boj2576.c
#include<stdio.h>

int main()
{
	int number;
	int sum=0;
	int min=100;

	for(int i=1;i<=7;i++)
	{
		scanf("%d",&number);
		if(number>=100||number<0)
			return 0;
		else if(number%2!=0)
		{
			sum+=number;
			if(number<min)
				min=number;
		}
	}
	if(sum!=0)
	{
		printf("%d\n",sum);
		printf("%d",min);
	}
	else
		printf("-1");
}

File: BOJ/boj2576.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {

		int[] num = new int[7];
		int sum = 0;
		int min = 101;

		Scanner scanner = new Scanner(System.in);

		for (int i = 0; i < num.length; i++) {
			num[i] = Integer.parseInt(scanner.nextLine());

			if (!((num[i] % 2) == 0)) {

				sum += num[i];
				if (num[i] < min) {
					min = num[i];
				}
			}
		}
		if (min == 101)
			System.out.println(-1);
		else
			System.out.printf("%d\n%d", sum, min);

		scanner.close();
		scanner = null;

	}

}
